services/ui-src/src/types/index.ts

```diff
@@ -181,6 +183,7 @@ export interface ReportContextShape extends ReportContextMethods {
 export enum ReportStatus {
   NOT_STARTED = "Not started",
   IN_PROGRESS = "In progress",
+  IN_REVISION = "In revision",
```

Comment:
Blocking: I think we decided against adding an `IN_REVISION` status because then the `update` handler, which always sets the state to `IN_PROGRESS`, would have to check each time to see if it had already been submitted. If I remember right, we were going to only check that on the dashboard, so if it had been submitted before and was in progress, the status would be `IN_PROGRESS` but we'd display "In revision".

Reply:
got it 👍🏽